event.json

```diff
@@ -0,0 +1,14 @@
+{
+  "pull_request": {
+    "number": 1,
+    "title": "Test PR",
+    "body": "This is a test pull request"
```

Comment:
The body of the pull request should provide more context or details about the changes or the reason for the pull request.